packages/backend/package.json

```diff
@@ -59,6 +61,8 @@
     "pg": "^8.7.1",
     "rate-limiter-flexible": "2.4.1",
     "react-markdown": "8.0.7",
+    "remark-breaks": "3.0.3",
+    "remark-gfm": "3.0.1",
```

Comment:
I think allowMultilineBreak doesn't work without `remark-breaks`, it gets a bit wonky and new lines become spaces only, for `remark-gfm` I believe I added it in just in case we needed to add like tables